logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/call/usecase/GetEstablishedCallsUseCase.kt

```diff
@@ -5,12 +5,12 @@ import com.wire.kalium.logic.feature.call.Call
 import com.wire.kalium.logic.sync.SyncManager
 import kotlinx.coroutines.flow.Flow
 
-class GetOngoingCallUseCase(
+class GetEstablishedCallsUseCase(
```

Comment:
```suggestion
class ObserveEstablishedCallsUseCase(
```

As we agreed on using the prefix `Observe` for Flows.